distribution/changelog.txt

```diff
@@ -4,6 +4,7 @@
 - Feature: [#21913] [Plugin] Allow precise and safe control of peep animations.
 - Improved: [#21981] Rendering performance of the map window has been improved considerably.
 - Improved: [#21981] The map window now defaults to showing as much of the map as fits the screen.
+- Improved: [#21983] Taking a screenshot now shows a 'success' message again, closing when taking another.
```

Comment:
No typewriter quotes, please. Also, since it also shows messages when failing, let’s leave it out entirely.

```suggestion
- Improved: [#21983] Taking a screenshot now shows a message again, closing when taking another.
```